src/LaravelPassport/Provider.php

```diff
@@ -79,7 +79,7 @@ protected function getUserByToken($token)
             ],
         ]);
 
-        return json_decode($response->getBody(), true);
+        return (array) json_decode($response->getBody(), true);
```

Comment:
I think I would prefer an explicit check for null vs a cast here.

Reply:
And if it is null what would you expect? An exception or the current behaviour of an empty array?

Reply:
@atymic ^ let me know what you'd prefer and I can update the PR.

Reply:
Actually, happy with this. Sorry for the delay :)